### ISSUE TITLE
Updated Subspace Index

### ISSUE THREAD

**Issue by jeroentskui** [alkem-io/client-web#6769]
**Is your feature request related to a problem? Please describe.**
Replace the current index view on subspaces with a view similar to the edit innovation flow view

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by SimoneZaza** [comment]
Good point to update this dialog. I do think we need to keep the ability to see the list without it being split across the innovation flow phases. However, it is possible to combine them I think ;)
